README.md

```diff
@@ -82,28 +91,28 @@ defmodule MyApp.Accounts do
   def create_user(attrs) do
     %User{}
     |> User.create_changeset(attrs)
-    |> Uploadex.create_with_file(MyUploader)
+    |> Uploadex.create_with_file()
   end
 
   def update_user(%User{} = user, attrs) do
     user
     |> User.update_changeset(attrs)
-    |> Uploadex.update_with_file(user, MyUploader)
+    |> Uploadex.update_with_file(user)
   end
 
   def delete_user(%User{} = user) do
     user
     |> Ecto.Changeset.change()
-    |> Uploadex.delete_with_file(MyUploader)
+    |> Uploadex.delete_with_file()
   end
 end
 ```
 
-For more flexibility, you can use the [Files](https://hexdocs.pm/uploadex/Uploadex.Files.html#content) module (or even the [arc functions](https://github.com/stavro/arc#basics)) directly.
+For more flexibility, you can use the [Files](https://hexdocs.pm/uploadex/Uploadex.Files.html#content) module directly.
 
 ## Motivation
 
-Even though there already exists a library to integrate Arc with Ecto (https://github.com/stavro/arc_ecto), this library was created because:
+Even though there already exists a library to for uploading files that integrates with ecto (https://github.com/stavro/arc_ecto), this library was created because:
```

Comment:
```suggestion
Even though there already exists a library for uploading files that integrates with ecto (https://github.com/stavro/arc_ecto), this library was created because:
```